ChangeLog

```diff
@@ -1,3 +1,15 @@
+Tag name: carma4_05
+Originator(s): C Bardeen
+Date: ???
+
+Gas ref indx and evap updates ???
```

Comment:
Not sure how to characterize these latest changes to CARMA base.

Reply:
Evap_mono and evap_poly are to fix a problem where a group that grows may have coremasses that don't evaporate back to another carma group (i.e. evap2elem is 0).

The other changes are to allow a gas to have a refractive index. This if for the optics table generation for groups that have water in equilibrium, but not as an actual element in the group (e.g. sulfates).